app/views/metadata_fields/_statutory_instruments.html.erb

```diff
@@ -17,3 +17,29 @@
       }
   %>
 <% end %>
+<%= render "shared/form_group", f: f, field: :primary_publishing_organisation do %>
+  <%= f.select :primary_publishing_organisation,
+      organisations_options,
+      {},
+      {
+        class: "select2 form-control",
+        multiple: false,
+        data: {
+          placeholder: "Select a primary publishing organisation"
+        }
+      }
+  %>
+<% end %>
+<%= render "shared/form_group", f: f, field: :organisations do %>
+  <%= f.select :organisations,
+      organisations_options,
+      {},
+      {
+        class: "select2 form-control",
+        multiple: true,
+        data: {
+          placeholder: "Select organisations"
```

Comment:
I think the terms we wanted in the editor itself were "Publishing organisation" and "Other associated organisations".